### ISSUE TITLE
watchdir is deprecated in minkNow 1.4.3

### ISSUE THREAD

**Issue by mafouille** [mattloose/RUscripts#3]
Hi Matt, 

Using aReadUntil is no longer compatible with how fast5 files are generated in online mode.
Indeed the option WATCHDIR (-w) allows to indicate the directory where the reads (fast5) are. Hence their analysis is done (counting the number of 2D reads, moving reads associated to a known amplicon).
In the "new" version of MinKnow (1.4.3), reads are moved in subdirectories. This is not allowed by the ReadUntil scripts.

```
$/opt/ONT/MinKNOW$ ls /var/lib/MinKNOW/data/reads/
0   100  103  106  109  111  14  17  2   22  25  28  30  33  36  39  41  44  47  5   52  55  58  60  63  66  69  71  74  77  8   82  85  88  90  93  96  99
1   101  104  107  11   12   15  18  20  23  26  29  31  34  37  4   42  45  48  50  53  56  59  61  64  67  7   72  75  78  80  83  86  89  91  94  97  tmp
10  102  105  108  110  13   16  19  21  24  27  3   32  35  38  40  43  46  49  51  54  57  6   62  65  68  70  73  76  79  81  84  87  9   92  95  98
```

I have no idea of the meaning of each subdirectory.

We tried to put the "Recursive = True" option on reading reads in aReadUntil but this does not change the results. Subdirectory reads are not parsed.
``` 
observer.schedule(event_handler, path=args.watchdir, recursive=True)
```

Martin

**Comment by mattloose** [comment]
Good point - I will look at this asap. 

The folder number are simply batches of reads. 

However I am not sure this will run correctly on R9.4 and minknow 1.43 at all at the minute. 

I'll get back to you on this.

**Comment by mattloose** [comment]
Yes - just to confirm. Switching the mode to recursive won't be sufficient to fix.  However there are other issues with read until at this moment in time (beyond these scripts). I will roll fixes together and post a fix as soon as possible.

**Comment by mattloose** [comment]
Actually - my comment above is incorrect. Switching the mode to recursive will fix it, but aReadUntil ignores reads that are already written to disk when you start running it. So it only looks at reads that are created whilst it is running.

The only outstanding issue to resolve is the moving of files - I am not actually quite sure how this behaves right now but will add to the check list.
